#request
int64 ready_go

---
#knfie point in real 3d world
geometry_msgs/Point beginPoint
